docker/prod/migrate-pictshare-to-pictrs.bash

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+set -e
+
+if [[ $(id -u) != 0 ]]; then 
+    echo "This migration needs to be run as root"
+    exit
+fi
+
+if [[ ! -f docker-compose.yml ]]; then 
+    echo "No docker-compose.yml found in current directory. Is this the right folder?"
+    exit
+fi
+
+# Fixing pictrs permissions
+mkdir -p volumes/pictrs
+sudo chown -R 991:991 volumes/pictrs
+
+echo "Restarting docker-compose, making sure that pictrs is started and pictshare is removed"
+docker-compose up -d --remove-orphans
+
+if [[ -z $(docker-compose ps | grep pictrs) ]]; then
+    echo "Pict-rs is not running, make sure you update Lemmy first"
+    exit
+fi
+
+if [[ -z $(type -P convert) ]]; then
+  echo "Installing imagemagick to convert .webp images to .jpg"
+  apt install imagemagick -y
+else 
+  echo "Imagemagick already installed."
+fi
```

Comment:
Pict-rs supports .webp now so this isnt needed anymore (and neither is the conversion below).